pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -99,10 +99,14 @@ usage_percentage=$(df -P "$target_dir" | awk 'NR==2 {print $5}' | sed 's/%%//')
 echo "[disk usage checker] Volume usage percentage: current = ${usage_percentage} ; allowed = ${usage_percentage_limit}"
 if [ "$usage_percentage" -gt "$usage_percentage_limit" ]; then
 	echo "[disk usage checker] Disk usage exceeds the volume percentage of ${usage_percentage_limit} for mounted directory, terminating..."
-	ps -o sess --no-headers | sort -u | while read sid; do
-		[[ "$sid" -eq "${$}" ]] && continue
-		pkill --signal SIGKILL --session "$sid"
-	done
+	if command -v setsid >/dev/null 2>&1 && command -v ps >/dev/null 2>&1 && command -v pkill >/dev/null 2>&1; then
```

Comment:
I would honestly just set a helper variable initially and then use it everywhere. This is getting ugly. Like, set a variable to non-empty value if we can do sessions, and do that at the beginning of `buildPodCommand`.

Reply:
I'm open to any suggestions as long as we can inject this helper variable from buildPodCommand to volumeCheckerScript. Would you please elaborate your exact suggestion?

Reply:
I think that everything is just concatenated in `buildPodCommand`, so isn't any variable visible later on? Perhaps you need to export it for the subshells to make it work...